test/topology_custom/test_tablets.py

```diff
@@ -23,3 +23,55 @@ async def test_tablet_change_replication_vnode_to_tablets(manager: ManagerClient
     with pytest.raises(InvalidRequest):
         await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
 
+
+@pytest.mark.asyncio
+async def test_tablet_change_replication_strategy(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
+    await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy'};")
+
+    res = await cql.run_async("DESCRIBE KEYSPACE test")
+    assert "NetworkTopologyStrategy" in res[0].create_statement, "NetworkTopologyStrategy wasn't switched onto"
+
+    res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
+    assert len(res) == 0, "tablets replication strategy turned on"
```

Comment:
This will test break the test as soon as we enable tablets by default (this test enshrines the fact they are off by default), so why even test this?

Reply:
Sure! So the PR enabling tablets by default will need to touch this test too. If you look at #16364 you'll see that a big part of this PRs is tuning existing tests here and there so that they don't crash after enabling tablets by default. One more test in this list is not a big deal